frontend/frontend/target_systems/__init__.py

```diff
@@ -0,0 +1,24 @@
+
+from contextlib import contextmanager
+from typing import Iterator
+from ._interface import TargetSystem, TargetSystemProvider
+from ._grpc import _GrpcTargetSystemProvider
+
+__all__ = ['TargetSystem', 'TargetSystemProvider', 'create_grpc_target_system_provider']
+
+
+@contextmanager
+def create_grpc_target_system_provider(server_address: str) -> Iterator[TargetSystemProvider]:
```

Comment:
I'm not super familiar with yields but I'm guessing it's called with something along the lines of
```python
with create_grpc_target_system_provider("server") as provider:
    provider.acquire_target_system("user","pass")
```

Is it then okay that every time we use it we only ask for one target system or we yield one and then discard the `provider` instance we got. I guess what I'm asking is this `_GrpcTargetSystemProvider` meant to act as a long running instance, or is it is fine either or?